src/components/welcome/MyAccounts/CreateButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@mui/material'
+import Link from 'next/link'
+import { AppRoutes } from '@/config/routes'
+import { useCurrentChain } from '@/hooks/useChains'
+
+const CreateButton = () => {
+  const currentChain = useCurrentChain()
+
+  return (
+    <Link
+      href={{ pathname: AppRoutes.newSafe.create, query: { chain: currentChain?.shortName } }}
```

Comment:
Should we also track this button click? IMO it should be the new entry point for safe creation flow cc @alicecha